Getting-Started-with-Akka-http-signature.md

```diff
@@ -0,0 +1,304 @@
+
+```scala
+@ val toTurtle = turtleWriter.asString(finalKeyPGGraph.graph,"").get
+
+toTurtle: String = """<#key>  <http://www.w3.org/ns/auth/cert#exponent>
+                65537 ;
+        <http://www.w3.org/ns/auth/cert#modulus>
+                "0087acb657366c6a911a0ce2470c24b8d85dd21ae76c6001db37d122c2eafe0fe2ae35541ccdd3c1c81603d98dd7d61b2c31c8605f81fbc3566604b7755793698836caaa99d868477c46b5b735529b50c0acfea6e10fc9b953697a67d57499801beb651e7e08343131a00d1873ab753ce5e79fc961874ee132472f7f210bc38966081fe263c620b67469b52cc555a26dee4fa10d8f40959e0e13516cc0bd1c1669ce53367d28248149142429b127f01e13d9bf21de52a4ac5694d5038a94178e144823b152fa19cbbc094dd40ddbd41e2195a19081125887fccedae21c50660c629a6f2ba4c85a54e1a6472f90b7b62ac7fe58fdae9daa0edc8edec49802fbfac7"^^<http://www.w3.org/2001/XMLSchema#hexBinary> .
+"""
+``` 
+This will return a String representation of the Pointed Graph in turtle format. This representation can also be saved on your local file system like so:
+
+```scala
+write(wd/"publicKey.ttl", RSAKeys.save(toTurtle))
+```
+
+----------------------------
+
+### Attaching Public keys to a File/URI
+
+One can use the `cp` or the `mv` Ammonite commands to move the public key file into the test_www directory which resides within the rww-play directory. The process of attaching the file, containing the key to a URI is very similar.
+
+```scala
+cp(wd/"publicKey.ttl" , wd/"rww-play"/"test_www"/"pubKey.ttl")
+```
+
+The Rww-play web server, which the Akka-http-signature library makes use of, uses symbolic links to indicate default representations of different documents. The Http content negotiation functionality is thus preserved. Http Content negotiation is a mechanism that is used for representing different representations of a resource at the same URI. The user can then specify which is best suited for them. 
+
+Content negotiation is preserved because the user can link to files without specifically addressing them via their original name by using these symbolic links. This makes it possible for links to be followed both on the file system and on the web in the same way. 
+
+This can be quite a complex mechanism and the setup for it can vary greatly between different servers.
+
+To create a symbolic link, one can make a symbolic link to their public key file using the following commands in the bash console:
+
+```bash
+$ ln -s pubKey.ttl pubKey
+```
+### Access Control
+
+Currently in order to manipulate the access control one can use `curl` and `PATCH` commands [As detailed in the rww-play wiki page](https://github.com/read-write-web/rww-play/wiki/Curl-Interactions).
+
+To demonstrate how access control changes user permitted actions, one can also manually edit the .acl.ttl file. To do this, one must navigate to the test_www directory (or the server that contains the Key files) and look for the correct file. After that in order to permit access to the , the user must comment out the following:
```

Comment:
"Comment out the following" is much too vague.

You comment out the commented line (In Turtle, comments are whatever follows a `#` sign), and you comment out the line that follows it `acl:agentClass` which gives everyone permission. Ie: you want to restrict persmission to yourself.

Please also remove all of the unnecessary extra white spaces here.